providers/openshift/provider.go

```diff
@@ -144,6 +144,7 @@ func (p *OpenShiftProvider) setCA(paths []string) error {
 		return err
 	}
 	p.Client.Transport = &http.Transport{
+		Proxy: http.ProxyFromEnvironment,
```

Comment:
We should modify the transport for this outside of the setCA function.

Reply:
http_proxy and https_proxy environment variables are handled by [http.DefaultTransport](https://golang.org/src/net/http/transport.go). This function is used to set custom CA on the transport but IMO it shouldn't discard other http.DefaultTransport default properties. But this is what is does since we need to change it later so it handle proxy env variables again.

I couldn't find a way to extend the p.client.transport outside this function without having to define the custom CA as well in the new function. 

Any idea how to do it?